Mathlib/Combinatorics/SetFamily/KruskalKatona.lean

```diff
@@ -290,5 +292,112 @@ theorem iterated_kk (h₁ : (𝒜 : Set (Finset (Fin n))).Sized r) (h₂ : 𝒞.
   · refine ih h₁.shadow (kruskal_katona h₁ h₂ h₃) ?_
     convert h₃.shadow
 
+/-- A special case of the **Kruskal-Katona theorem** which is sometimes easier to work with.
+
+If `|𝒜| ≥ k choose r`, (and everything in `𝒜` has size `r`) then the initial segment we compare to
+is just all the subsets of `{0, ..., k - 1}` of size `r`. The `i`-th iterated shadow of this is all
+the subsets of `{0, ..., k - 1}` of size `r - i`, so the `i`-th iterated shadow of `𝒜` has at least
+`k.choose (r - i)` elements. -/
+theorem lovasz_form (hir : i ≤ r) (hrk : r ≤ k) (hkn : k ≤ n)
```

Comment:
I'm not entirely convinced this is a great name either - this is a special case of the Lovasz form, and that's not even a particularly helpful theorem name. Certainly the docstring should mention that this is Lovasz' formulation and emphasise that over the Kruskal-Katona name, however.

Reply:
Let me know how you like the new name and docstring